lib/csv_class_maker/csv_find.rb

```diff
@@ -70,9 +94,16 @@ def each
       end
     end
 
-
     private
 
+    def extract_headers(file_name, options)
+      csv_file = File.open(file_name,'r')
+
+      @@headers ||= CSV.new(csv_file, options).first.map do |headers, values|
```

Comment:
Unused block argument - `values`. If it's necessary, use `_` or `_values` as an argument name to indicate that it won't be used.